alleycats-core/src/main/scala/alleycats/std/set.scala

```diff
@@ -65,6 +69,10 @@ trait SetInstances {
         go(f(a))
         bldr.result()
       }
+
+      override def empty[A]: Set[A] = Set.empty
+
+      override def combineK[A](x: Set[A], y: Set[A]): Set[A] = x | y
```

Comment:
These are effectively re-implemented from `catsStdInstancesForSet` in `cats-core`:

https://github.com/typelevel/cats/blob/3d900dedf121223debec27207a13cbd374381235/core/src/main/scala/cats/instances/set.scala#L23-L25

...I wasn't sure if delegating to the `cats-core` typeclass instance was the thing to do, or if it was clearer just to duplicate!

Reply:
I'm okay with this.  The duplication is slight and the laws ought to catch problems.